modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -43,10 +43,16 @@ You can use any of the following options for authentication and authorization wh
 
 ** Using <<create-api-ASM-CLI, user credentials>> (username and password)
 +
-This option supports only those users that are managed by Anypoint Platform.
+This option supports only those users that are:
```

Comment:
Typically we'd say either "only those users who are" (slight preference for referring to people with "who" and objects, plants, animals with "that").

Reply:
Done! Thanks!